.github/workflows/openapi-check.yml

```diff
@@ -21,5 +24,9 @@ jobs:
         uses: ./.github/actions/prepare-backend
       - name: Generate version.properties
         run: ./bin/build.sh :steps [:version]
+      - name: Check and commit OpenAPI changes
+        if: github.event_name == 'pull_request'
+        run: ./bin/openapi-check.sh --auto-commit
```

Comment:
I think this will only auto-commit on PR open, not when a PR is updated with a new commit